weather.go

```diff
@@ -25,26 +27,23 @@ type Weather struct {
 		Pressure  int     `json:"pressure"`
 		Humidity  int     `json:"humidity"`
 	} `json:"main"`
+	Name string `json:"name"`
 }
 
 type CityUnknown struct {
 	Cod     string `json:"cod"`
 	Message string `json:"message"`
 }
 
-type CliResponse struct {
+type Conditions struct {
+	City    string
 	OneWord string  `json:"oneword"`
```

Comment:
Do we ever marshal this struct to JSON? If not, we don't need to supply `json` tags for it. (And if we do, these will be the default JSON field names anyway. You only need to supply a tag if the JSON field names are different from the Go field names.)